examples/python/brand/Monda-OFL.txt

```diff
@@ -0,0 +1,93 @@
+Copyright 2021 The Monda Project Authors (https://github.com/googlefonts/mondaFont)
```

Comment:
(Commenting here because I can't comment on the .ttf file directly.)
Something to keep in mind for future examples: All of the examples right now are bundled into a single file, examples.json, and that whole file must be loaded before the page starts running an example. So that's a reason to try to example file sizes small. In the future, it might be better to put each example in its own .json file that is loaded on demand. I don't think it's a show stopper here, but it is something to keep in mind in the future when adding large examples.

Reply:
I included this in part to test out the full range of features. Having tested it, I'm on the fence. I'd be okay taking it out for simplicity and sizing; or I'd equally be okay with leaving it so that we continue to test it manually before py-shiny releases. It adds about 200kb to `examples.json`, which is ~400kb before this change and just over 600kb after.